### PR TITLE
satoshis -> zatoshis

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Review by daira — CHANGES_REQUESTED** [review]
The JSON RPC outputs can't be changed for compatibility reasons. We could have both "satoshis" and "zatoshis" fields, I guess, but I'd rather just not change them.

Everything else looks good.

**Comment by rex4539** [comment]
Could you please pinpoint the exact cases that need to be reverted? I have no idea :)

File: qa/rpc-tests/addressindex.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def check_balance(node_index, address, expected_balance, expected_received=None)
             expected += i + 1
             expected_deltas.append({
                 'height': 106 + i,
-                'satoshis': (i + 1) * COIN,
+                'zatoshis': (i + 1) * COIN,
```
Comment (daira) [inline qa/rpc-tests/addressindex.py:157]: Revert this and other renames involving outputs of RPC calls.

File: qa/rpc-tests/spentindex.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def run_test(self):
         assert_equal(len(coinbase_tx['outputs']), 2)
         assert_equal(coinbase_tx['outputs'][0]['index'], 0)
         assert_equal(coinbase_tx['outputs'][1]['index'], 1)
-        assert_equal(coinbase_tx['outputs'][1]['satoshis'], 2.5*COIN)
+        assert_equal(coinbase_tx['outputs'][1]['zatoshis'], 2.5*COIN)
```
Comment (daira) [inline qa/rpc-tests/spentindex.py:134]: Revert this and other renames involving outputs of RPC calls (including everything in this file).

File: src/rpc/blockchain.cpp [hunk 3]
```diff
@@ -168,7 +168,7 @@ UniValue blockToDeltasJSON(const CBlock& block, const CBlockIndex* blockindex)
                 if (IsValidDestination(dest)) {
                     delta.pushKV("address", keyIO.EncodeDestination(dest));
                 }
-                delta.pushKV("satoshis", -1 * spentInfo.satoshis);
+                delta.pushKV("zatoshis", -1 * spentInfo.zatoshis);
```
Comment (daira) [inline src/rpc/blockchain.cpp:171]: Revert this and other renames involving outputs of RPC calls (including everything in this file).

File: src/rpc/mining.cpp [hunk 4]
```diff
@@ -374,8 +374,8 @@ UniValue prioritisetransaction(const UniValue& params, bool fHelp)
             "1. \"txid\"       (string, required) The transaction id.\n"
             "2. priority delta (numeric, required) The priority to add or subtract.\n"
             "                  The transaction selection algorithm considers the tx as it would have a higher priority.\n"
-            "                  (priority of a transaction is calculated: coinage * value_in_satoshis / txsize) \n"
-            "3. fee delta      (numeric, required) The fee value (in satoshis) to add (or subtract, if negative).\n"
+            "                  (priority of a transaction is calculated: coinage * value_in_zatoshis / txsize) \n"
+            "3. fee delta      (numeric, required) The fee value (in zatoshis) to add (or subtract, if negative).\n"
```
Comment (daira) [inline src/rpc/mining.cpp:378]: These are fine.

File: src/rpc/misc.cpp [hunk 5]
```diff
@@ -615,7 +615,7 @@ UniValue getaddressmempool(const UniValue& params, bool fHelp)
             "    \"address\"  (string) The base58check encoded address\n"
             "    \"txid\"  (string) The related txid\n"
             "    \"index\"  (number) The related input or output index\n"
-            "    \"satoshis\"  (number) The difference of zatoshis\n"
+            "    \"zatoshis\"  (number) The difference of zatoshis\n"
```
Comment (daira) [inline src/rpc/misc.cpp:618]:
```suggestion
            "    \"satoshis\"  (number) The difference of zatoshis (note 'satoshis' spelling of this field)\n"
```

File: src/rpc/misc.cpp [hunk 6]
```diff
@@ -697,7 +697,7 @@ UniValue getaddressutxos(const UniValue& params, bool fHelp)
             "    \"height\"  (number) The block height\n"
             "    \"outputIndex\"  (number) The output index\n"
             "    \"script\"  (string) The script hex encoded\n"
-            "    \"satoshis\"  (number) The number of zatoshis of the output\n"
+            "    \"zatoshis\"  (number) The number of zatoshis of the output\n"
```
Comment (daira) [inline src/rpc/misc.cpp:700]:
```suggestion
            "    \"satoshis\"  (number) The number of zatoshis of the output (note 'satoshis' spelling of this field)\n"
```

File: src/rpc/misc.cpp [hunk 7]
```diff
@@ -710,7 +710,7 @@ UniValue getaddressutxos(const UniValue& params, bool fHelp)
             "        \"height\"      (number)  The block height\n"
             "        \"outputIndex\" (number)  The output index\n"
             "        \"script\"      (string)  The script hex encoded\n"
-            "        \"satoshis\"    (number)  The number of zatoshis of the output\n"
+            "        \"zatoshis\"    (number)  The number of zatoshis of the output\n"
```
Comment (daira) [inline src/rpc/misc.cpp:713]:
```suggestion
            "        \"satoshis\"    (number)  The number of zatoshis of the output (note 'satoshis' spelling of this field)\n"
```

File: src/rpc/misc.cpp [hunk 8]
```diff
@@ -760,7 +760,7 @@ UniValue getaddressutxos(const UniValue& params, bool fHelp)
         output.pushKV("txid", it.first.txhash.GetHex());
         output.pushKV("outputIndex", (int)it.first.index);
         output.pushKV("script", HexStr(it.second.script.begin(), it.second.script.end()));
-        output.pushKV("satoshis", it.second.satoshis);
+        output.pushKV("zatoshis", it.second.zatoshis);
```
Comment (daira) [inline src/rpc/misc.cpp:763]: Revert this.

File: src/rpc/misc.cpp [hunk 9]
```diff
@@ -853,7 +853,7 @@ UniValue getaddressdeltas(const UniValue& params, bool fHelp)
             "\nResult:\n"
             "[\n"
             "  {\n"
-            "    \"satoshis\"  (number) The difference of zatoshis\n"
+            "    \"zatoshis\"  (number) The difference of zatoshis\n"
```
Comment (daira) [inline src/rpc/misc.cpp:856]:
```suggestion
            "    \"satoshis\"  (number) The difference of zatoshis (note 'satoshis' spelling of this field)\n"
```

File: src/rpc/misc.cpp [hunk 10]
```diff
@@ -865,7 +865,7 @@ UniValue getaddressdeltas(const UniValue& params, bool fHelp)
             "  \"deltas\":\n"
             "    [\n"
             "      {\n"
-            "        \"satoshis\"    (number) The difference of zatoshis\n"
+            "        \"zatoshis\"    (number) The difference of zatoshis\n"
```
Comment (daira) [inline src/rpc/misc.cpp:868]:
```suggestion
            "        \"satoshis\"    (number) The difference of zatoshis (note 'satoshis' spelling of this field)\n"
```

File: src/rpc/misc.cpp [hunk 11]
```diff
@@ -921,7 +921,7 @@ UniValue getaddressdeltas(const UniValue& params, bool fHelp)
         delta.pushKV("blockindex", (int)it.first.txindex);
         delta.pushKV("height", it.first.blockHeight);
         delta.pushKV("index", (int)it.first.index);
-        delta.pushKV("satoshis", it.second);
+        delta.pushKV("zatoshis", it.second);
```
Comment (daira) [inline src/rpc/misc.cpp:924]: Revert this.

File: doc/release-notes/release-notes-3.1.0-rc1.md [hunk 12]
```diff
@@ -51,7 +51,7 @@ Alfredo Garcia (28):
       add estimatedheight to getblockchaininfo
       add documentation and command line parsing to afl scripts
       get fuzzing options from directory
-      add bool argument to get balance in satoshis to rpc getreceivedbyaddress
+      add bool argument to get balance in zatoshis to rpc getreceivedbyaddress
```
Comment (daira) [inline doc/release-notes/release-notes-3.1.0-rc1.md:54]: Revert release notes changes (so that it's easy to search commits by these messages).

**Comment by rex4539** [comment]
Ok, not worth it. Sorry for wasting your time :)

**Pull request closed without merging**